components/ResultsPage/YourAnswers.tsx

```diff
@@ -17,6 +17,28 @@ export const YourAnswers: React.VFC<{
     tsln._language
   )
 
+  /**
+   * shouldDisplay
+   *    Returns  False  when Income Provided is Yes or OAS deferral is Delay
+   *    Returns  True   for any other scenario
+   */
+  function shouldDisplay(input: FieldInput): boolean {
```

Comment:
Seems to work as expected but may I suggest rewriting the "shouldDisplay" function? Currently it's a bit hard to read and I don't think you need the first check? I tried this and it works as well:

```
function shouldDisplay(input: FieldInput): boolean {
    const dontShow: String[] = ['incomeAvailable', 'partnerIncomeAvailable', 'oasDefer']
    return !dontShow.includes(input.key) && input.value !== 'true'
 }
```